### PR TITLE
Docs: [AEA-4540] -update datetime for 1day prescriptions

### PR DESCRIPTION
**Pull request by jonathanwelch1-nhs**
## Summary

Medicus were trying to create / cancel 1 day prescriptions and found that if they specify the same end date as start date for a prescription (should be valid) then the FHIR validation considered it invalid. The recommendation to them was to specify 23:59:59 on the end date but the documentation could still lead other suppliers to the same issue.

- Routine Change

### CODE REVIEW

**Review by MatthewPopat-NHS — CHANGES_REQUESTED** [review]
We also need to update the files in the examples folder.

File: packages/specification/schemas/components/MedicationRequest/DispenseNotificationMedicationRequest.yaml [hunk 1]
```diff
@@ -160,8 +160,9 @@ properties:
             description: |
               End date of an issue; capped at 12 months and cannot exceed the prescription's end date.
               Overall validity of the drug.
-            format: date-time
-            example: "2022-10-21T23:59:59Z"
+              Must be in date-time format with the time set to 23:59:59 on the last valid day of the prescription, including for single-day prescriptions. 
+              Example: "2022-10-21T23:59:59Z"
```
Comment (seansteberisal) [inline packages/specification/schemas/components/MedicationRequest/DispenseNotificationMedicationRequest.yaml:164]: I think example: here needs a slight unindent?

File: packages/specification/schemas/components/MedicationRequest/MedicationRequest.yaml [hunk 2]
```diff
@@ -156,8 +156,9 @@ properties:
             description: |
               End date of an issue; capped at 12 months and cannot exceed the prescription's end date.
               Overall validity of the drug.
+              Must be in date-time format with the time set to 23:59:59 on the last valid day of the prescription, including for single-day prescriptions. 
+              Example: "2022-10-21T23:59:59Z"
```
Comment (seansteberisal) [inline packages/specification/schemas/components/MedicationRequest/MedicationRequest.yaml:160]: I think example: here needs a slight unindent?

File: packages/specification/schemas/components/MedicationRequest/ReleaseMedicationRequest.yaml [hunk 3]
```diff
@@ -153,8 +153,9 @@ properties:
             description: |
               End date of an issue; capped at 12 months and cannot exceed the prescription's end date.
               Overall validity of the drug.
+              Must be in date-time format with the time set to 23:59:59 on the last valid day of the prescription, including for single-day prescriptions. 
+              Example: "2022-10-21T23:59:59Z"
```
Comment (seansteberisal) [inline packages/specification/schemas/components/MedicationRequest/ReleaseMedicationRequest.yaml:157]: I think example: here needs a slight unindent?

**Review by seansteberisal — CHANGES_REQUESTED** [review]

**Review by seansteberisal — DISMISSED** [review]

**Review by MatthewPopat-NHS — APPROVED** [review]

**Pull request merged**